From 632fba5070370818ab5dca737ea7fbb4af35d6ab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakub=20Jank=C5=AF?= <janku.jakub.jj@gmail.com>
Date: Tue, 5 Dec 2017 19:45:53 +0100
Subject: [PATCH 53/55] x11: update wms_with_icons_on_desktop[] to support Mate

Since version 1.9.2, Marco sets _NET_WM_NAME to "Metacity (Marco)"
instead of "Marco", see:

http://git.mate-desktop.org/marco/commit/?id=cbf24f5d78719b930fc0cd6c0c80e00e9d62c95a

Add this new value to wms_with_icons_on_desktop[], so that
vdagent_x11_has_icons_on_desktop() returns TRUE.

Acked-by: Victor Toso <victortoso@redhat.com>
---
 src/vdagent/x11.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vdagent/x11.c b/src/vdagent/x11.c
index 61cc4c7..9700847 100644
--- a/src/vdagent/x11.c
+++ b/src/vdagent/x11.c
@@ -1296,6 +1296,7 @@ int vdagent_x11_has_icons_on_desktop()
         "Metacity", /* GNOME-2 or GNOME-3 fallback */
         "Xfwm4",    /* Xfce */
         "Marco",    /* Mate */
+        "Metacity (Marco)", /* Mate, newer */
         NULL
     };
     const gchar *net_wm_name = vdagent_x11_get_wm_name();
-- 
2.16.1

